content/pages/download-va-letters.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Download VA Letters
+layout: page-breadcrumbs.html
+---
+<script src="https://standards.usa.gov/assets/js/vendor/uswds.min.js" type="text/javascript"></script>
+
+<div class="main">
+<div class="section">
```

Comment:
According to the guidance provided in `navigation-links-list.html` we may want wrap each part with a heading into its own section, and replace the `<div class="section">` here with `<section>` and `<div class="main">` with `<main>`.